system/jlib/jdebug.cpp

```diff
@@ -947,20 +952,20 @@ memsize_t getMapInfo(const char *type)
         bool skipline = true;
         if ( !strcmp(type, "heap") ) // 'general' heap includes anon mmapped + sbrk
         {
-            if ( strstr(ln, "[heap]") || (!strstr(ln, " /") && !strstr(ln, " [")) )
+            if ( strstr(ln, "[heap") || (!strstr(ln, " /") && !strstr(ln, " [")) )
```

Comment:
Not sure why this and similar diffs to "[stack]"->"[stack" are showing..
Does it need rebasing onto commit from HPCC-15416 ?